README.md

```diff
@@ -41,35 +48,36 @@ try:
 	# Iterate through the recordings in the library.
 	for recording in library:
 
+		videofilename = datetime.datetime.fromtimestamp(int(recording['name'])//1000).strftime('%Y-%m-%d %H-%M-%S') + ' ' + recording['uniqueId']
 		##
 		# The videos produced by Arlo are pretty small, even in their longest, best quality settings,
 		# but you should probably prefer the chunked stream (see below). 
 		###    
 		#    # Download the whole video into memory as a single chunk.
 		#    video = arlo.GetRecording(recording['presignedContentUrl'])
-		#	 with open(recording['name']+'.mp4', 'w') as f:
+		#	 with open(videofilename+'.mp4', 'w') as f:
```

Comment:
Minor tweak, but maybe we should append .mp4 to videofilename on line 51 where you assign videofilename so we don't have to keep appending it.